content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Upgrade with Helm
+linktitle: Upgrade with Helm
+description: Upgrade and configure Istio for in-depth evaluation.
+weight: 27
+keywords: [kubernetes,helm]
+owner: istio/wg-environments-maintainers
+icon: helm
+test: no
+---
+
+Follow this guide to upgrade and configure an Istio mesh using
+[Helm](https://helm.sh/docs/) for in-depth evaluation.  This guy assumes you have already performed an
+[installation with Helm](../../install/helm) for a previous minor or patch version of Istio.
+
+{{< boilerplate helm-preamble >}}
+
+{{< boilerplate helm-hub-tag >}}
+
+{{< boilerplate helm-prereqs >}}
+
+## Upgrade steps
+
+Change directory to the root of the release package and then
+follow the instructions below.
+
+{{< boilerplate helm-jwt-warning >}}
+
+### Create a backup
+
+{{< boilerplate helm-backup >}}
+
+### Migrating from non-Helm installations
```

Comment:
Would you migrate from a non-helm installation during upgrade? Also the template say to `re-install Istio using Helm as described above` which isn't true anymore (it's on another page).

Reply:
Removed this section from the upgrade template. I agree with you.